content/_code-samples/freeze/py/unfreeze_token.py

```diff
@@ -0,0 +1,49 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import IssuedCurrencyAmount, TrustSet
+from xrpl.transaction import (safe_sign_and_autofill_transaction,
+                              send_reliable_submission)
+from xrpl.models import TrustSetFlag
+from xrpl.utils import str_to_hex
+from xrpl.wallet import generate_faucet_wallet
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # connect to testnet
+
+token_name = "USD"
+
+# Amount a trustline can handle
+value = "0"
+
+# Address to unfreeze trustline
+target_addr = generate_faucet_wallet(client=client).classic_address
+
+# Sender wallet
+sender_wallet = generate_faucet_wallet(client=client)
```

Comment:
One last print statement I'd recommend is before any wallet generation - especially in the files generating two wallets, because it currently seems to be doing nothing for 8-10 seconds as it tries to generate these wallets. 
```suggestion
print("Generating two test wallets...")

# Address to unfreeze trustline
target_addr = generate_faucet_wallet(client=client).classic_address

# Sender wallet
sender_wallet = generate_faucet_wallet(client=client)
```